src/handlers/shows.js

```
@@ -6,13 +6,18 @@ import {
   ADD_SHOW,
   ALREADY_WANTED,
   NO_SHOW_FOUND,
-  NO_SHOW_QUEUED
+  NO_SHOW_QUEUED,
+  NO_SHOW_SLOT
 } from '~/responses/shows.js';
 
 export async function handleFindShowIntent(req, resp) {
   const api = getProvider(PROVIDER_TYPE.SHOWS);
   const showName = req.slot('showName');
 
+  if (!showName) {
+    return Promise.resolve(resp.say(NO_SHOW_SLOT()));
```

This should throw an error. Check out the movies.js file